spec.bs

```diff
@@ -410,19 +410,20 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
     1. Run [=interest group update=] with |auctionConfig|'s
       [=auction config/interest group buyers=].
 1. Let |queue| be the result of [=starting a new parallel queue=].
+1. Let |reportUrls| be a new [=list/is empty|empty=] [=list=].
 1. [=parallel queue/enqueue steps|Enqueue the following steps=] to |queue|:
   1. Let |winner| be the result of running [=generate and score bids=] with |auctionConfig|, null,
-    and |global|.
+    |reportUrls|, and |global|.
   1. If |winner| is null:
     1. [=Queue a global task=] on [=DOM manipulation task source=], given |global|, to resolve |p|
       with null.
   1. Otherwise:
-    1. [=Queue a global task=] on the [=DOM manipulation task source=], given |global|, to resolve |p|
-
-      with |winner|'s [=generated bid/ad descriptor=]. TODO: resolve |p| with urn-uuid, instead of a
-      URL.
+    1. [=Queue a global task=] on the [=DOM manipulation task source=], given |global|, to resolve
+      |p| with |winner|'s [=generated bid/ad descriptor=]. TODO: resolve |p| with urn-uuid, instead
+      of a URL.
   1. Run [=interest group update=] with |auctionConfig|'s [=auction config/interest group buyers=].
   1. TODO: Update bidCount and prevWins for interest groups that participated in the auction.
+  1. TODO: Fetch URLs of |reportUrls| when the winning ad is navigated to.

```

Comment:
wonderring how/where to spec this (mostly not clear about "the winning ad is navigated to" part).

Reply:
Protected Audience (this spec) doesn't do the fetches, we put the URLs in the `FencedFrameConfig` and the Fenced Frame does the fetching.  See the "TODO: Set URLs passed to sendReportTo() into the FencedFrameConfig so they can be fetched when ad renders."

Reply:
FencedFrameConfig includes sendReportTo URLs (reportWin and reportResult)? I thought FencedFrameConfig only reports the beacons registered with "reportEvent()", and PA reports.

Reply:
Left the FF config thing as TODOs, and should be in a separate PR. Our spec doesn't have fenced frame config, and seems the FF spec does not have callback fields for fetching these URLs.